### PR TITLE
Follow the Wordpress Online Store standard

### PR DESCRIPTION
**Pull request by guzzilar**
## PR Purpose
- Added `readme.txt` file and changed file name (`omise-gateway.php` => `omise-woocommerce.php`) for follow the Wordpress Online Store standard.
- Fixed links in the `README.md` file

### CODE REVIEW

**Comment by guzzilar** [comment]
FYI: Tested on WooCommerce/2.4.10, WordPress/4.3.1

File: omise-woocommerce.php [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
-defined('ABSPATH') or die("No direct script access allowed.");
-
 /**
- * Plugin Name: Omise Gateway Wordpress plugin
- * Plugin URI: http://docs.omise.co/omise-wp
- * Description: Allows easy integrating the Omise Payment gateway
- * Version: 1.0.2
- * Author: Omise Team
+ * Plugin Name: Omise Payment Gateway
+ * Plugin URI: https://docs.omise.co/woocommerce
```
Comment (fred) [inline omise-woocommerce.php]: `https://www.omise.co/woocommerce-plugin`

Comment (guzzilar) [inline omise-woocommerce.php]: :bow:

**Comment by fred** [comment]
:+1:

**Pull request merged**